runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -560,6 +559,22 @@ else if (nextSuperAntifireTick - tickCount <= 0)
 		{
 			updateVarTimer(FARMERS_AFFINITY, event.getValue(), i -> i * 20);
 		}
+
+		if (event.getVarbitId() == Varbits.PRAYER_ENHANCE && config.showPrayerEnhance())
+		{
+			if (event.getValue() == 80)
+			{
+				createGameTimer(PRAYER_ENHANCE);
+			}
+			if (event.getValue() == 0)
+			{
+				removeGameTimer(PRAYER_ENHANCE);
+			}
```

Comment:
You should only ever call `updateVarTimer()`, not `createGameTimer`/`removeGameTimer` when using var-based timers; that function will handle creation, update, and deletion of the timer.